word_guess.rb

```diff
@@ -0,0 +1,269 @@
+      puts pac_man_fiv + dot_fiv + dot_fiv + ghost_fiv.colorize(:red) + ghost_fiv.colorize(:green)
+      puts dot_six + dot_six + ghost_six + ghost_six + pac_man_six
+      puts @word_so_far
+      puts "The letters you have guessed so far are: #{@used_letters}"
+
+    when 2
+      puts ghost_one + ghost_one + ghost_one + pac_man_one + dot_one
+      puts pac_man_two + dot_two + ghost_two.colorize(:blue) + ghost_two.colorize(:red) + ghost_two.colorize(:green)
+      puts pac_man_thr + dot_thr + ghost_thr.colorize(:blue) + ghost_thr.colorize(:red) + ghost_thr.colorize(:green)
+      puts pac_man_fou + dot_fou + ghost_fou.colorize(:blue) + ghost_fou.colorize(:red) + ghost_fou.colorize(:green)
+      puts pac_man_fiv + dot_fiv + ghost_fiv.colorize(:blue) + ghost_fiv.colorize(:red) + ghost_fiv.colorize(:green)
+      puts ghost_six + ghost_six + ghost_six + pac_man_six + dot_six
+      puts @word_so_far
+      puts "The letters you have guessed so far are: #{@used_letters}"
+
+    when 1
+      puts pac_man_one + ghost_one + ghost_one + ghost_one + ghost_one
+      puts pac_man_two + ghost_two.colorize(:magenta) + ghost_two.colorize(:blue) + ghost_two.colorize(:red) + ghost_two.colorize(:green)
+      puts pac_man_thr + ghost_thr.colorize(:magenta) + ghost_thr.colorize(:blue) + ghost_thr.colorize(:red) + ghost_thr.colorize(:green)
+      puts pac_man_fou + ghost_fou.colorize(:magenta) + ghost_fou.colorize(:blue) + ghost_fou.colorize(:red) + ghost_fou.colorize(:green)
+      puts pac_man_fiv + ghost_fiv.colorize(:magenta) + ghost_fiv.colorize(:blue) + ghost_fiv.colorize(:red) + ghost_fiv.colorize(:green)
+      puts pac_man_six + ghost_six + ghost_six + ghost_six + ghost_six
+      puts @word_so_far
+      puts "The letters you have guessed so far are: #{@used_letters}"
+
+    when 0
+      puts ghost_one + ghost_one + ghost_one + ghost_one + ghost_one
+      puts ghost_two.colorize(:yellow).blink + ghost_two.colorize(:magenta).blink + ghost_two.colorize(:blue).blink + ghost_two.colorize(:red).blink + ghost_two.colorize(:green).blink
+      puts ghost_thr.colorize(:yellow).blink + ghost_thr.colorize(:magenta).blink + ghost_thr.colorize(:blue).blink + ghost_thr.colorize(:red).blink + ghost_thr.colorize(:green).blink
+      puts ghost_fou.colorize(:yellow).blink + ghost_fou.colorize(:magenta).blink + ghost_fou.colorize(:blue).blink + ghost_fou.colorize(:red).blink + ghost_fou.colorize(:green).blink
+      puts ghost_fiv.colorize(:yellow).blink + ghost_fiv.colorize(:magenta).blink + ghost_fiv.colorize(:blue).blink + ghost_fiv.colorize(:red).blink + ghost_fiv.colorize(:green).blink
+      puts ghost_six + ghost_six + ghost_one + ghost_six + ghost_six
+      puts @word_so_far
+
+    else
+      puts "I don't know what's going on!"
+      exit
+
+    end
+  end
+end
+
+game = WordGuess.new
+puts "Welcome to the Pac-Man Word Guess! Please play starting now!"
+game.play_the_game
```

Comment:
I like the way you've encapsulated the all of the game functionality in the `play_the_game` method.